src/components/Admin/custom/PostPreview.tsx

```diff
@@ -23,7 +23,7 @@ export const PostPreview: FC = () => {
                   caption={formData?.caption ?? ''}
                   short_text={formData?.short_text ?? ''}
                   details={formData?.details ?? ''}
-                  added_at={formData?.added_at ?? ''}
+                  added_at={formData?.added_at ?? new Date().toISOString()}
```

Comment:
rozmyslam, ze mozno tie added_at inputy schovame 🤔 ukazovat tam toto mi pride off, lebo hodnota toho fieldu sa aj tak (z nejakeho dovodu) generuje automaticky, tak ked pises post 5 minut, tak to vlastne nie je pravda 😄 ale neviem, nezalezi na tom